.github/workflows/packages.yml

```diff
@@ -0,0 +1,37 @@
+name: packages
+
+on:
+  workflow_call: ~
+  pull_request:
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.asciidoc'
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+      - '**/*.asciidoc'
+  schedule:
+    - cron: '0 2 * * *'
```

Comment:
What's the reason for this?

Reply:
In Jenkins there was a nightly job that runs the full matrix. Here it's solved with this: https://github.com/elastic/apm-agent-python/pull/1731#discussion_r1094483988

Reply:
Okay.. I just noticed it's the wrong file and a copy pasta error.

Reply:
https://github.com/elastic/apm-agent-python/pull/1731/commits/e368f55bdfc5a59fa647df4c8296d28b3fa95355